### DISCUSSION TITLE
Exclude ads domain

### DISCUSSION THREAD

**Discussion by SemutJahat** [coskundeniz/google-ad-clicker-premium#7] [Ideas]
hey, is there any feature to do excluding exact domain from clicked using this tool?

example : 
1. ads-domain1.com
2. ads-domain2.com
3. **_ads-domain3.com_**
4. ads-domain4.com
5. ads-domain5.com

and there is a feature to exclude ads-domain3.com to be clicked

**Comment by coskundeniz** [comment]
Yes, there is **excludes** option. You can read the explanation for it.
